addons/sale/views/sale_portal_templates.xml

```diff
@@ -862,7 +870,7 @@
                                         </td>
                                         <td name="td_priceunit" class="text-end text-nowrap">
                                             <span
-                                                t-out="section_line['price_subtotal']"
+                                                t-out="section_line[price_field]"
```

Comment:
The unit price should always be the untaxed amount when considering grouped lines no?
I didn't use the `price_field` there on purpose.